### DISCUSSION TITLE
See the actual page that someone clicked on my website from

### DISCUSSION THREAD

**Discussion by churichard** [plausible/analytics#1102] [Anything Else]
I think it would be nice to be able to see the exact page that someone clicked on my website from. So for example, if I see that Twitter is a source, I would love to be able to click on it and drill down to the exact tweets that drove the most traffic. Same with other websites such as Reddit, Hacker News, etc.

I know Simple Analytics actually has support for this for Twitter - they let you see the exact tweets that drove traffic to your website.

**Comment by metmarkosaric** [comment]
we do this but all the browsers including Chrome, Safari and Firefox have stopped sending this data in March this year. you can read more here https://plausible.io/blog/referrer-policy

we do try to display some tweets but [the API connection](https://plausible.io/docs/twitter) to them is not working the best. you need to click on the Details button after drilling down into Twitter to see what we can find. see an example from [our live demo](https://plausible.io/plausible.io/referrers/Twitter?period=month&source=Twitter&date=2021-05-01)

there's this idea too #873
